load/load_aix_nocgo.go

```diff
@@ -20,7 +21,7 @@ func AvgWithContext(ctx context.Context) (*AvgStat, error) {
 		return nil, err
 	}
 
-	idx := strings.Index(string(line), "load average:")
+	idx := bytes.Index(line, []byte("load average:"))
```

Comment:
Was it reported by gocritic? It looks alike something reported by `mirror` linter